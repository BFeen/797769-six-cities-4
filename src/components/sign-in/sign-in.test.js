import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import SignIn from "./sign-in.jsx";


Enzyme.configure({
  adapter: new Adapter(),
});

it(`SignIn rendering correctly`, () => {
  const tree = shallow(
      <SignIn
        onSubmit={() => {}}
      />
  );

  expect(tree).toMatchSnapshot();
});
