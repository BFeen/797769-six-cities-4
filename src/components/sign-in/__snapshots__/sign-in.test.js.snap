// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn rendering correctly 1`] = `ShallowWrapper {}`;
